assets/js/util/index.js

```diff
@@ -714,28 +714,49 @@ export const deleteCache = ( cacheType, cacheKey ) => {
 	return true;
 };
 
-export const findTagInIframeContent = ( iframe, module ) => {
+export const findTagInHtmlContent = ( html, module ) => {
 	let existingTag = false;
 
-	if ( ! iframe ) {
+	if ( ! html ) {
 		return false;
 	}
 
-	// Check if tag present in <head>.
-	const head = iframe.contentWindow.document.querySelector( 'head' );
-	if ( head ) {
-		existingTag = extractTag( head.innerHTML, module );
-	}
+	existingTag = extractTag( html, module );
+
+	return existingTag;
+};
+
+/**
+ * Looks for existing tag requesting front end html, if no existing tag was found on server side
+ * while requesting list of accounts.
+ *
+ * @param {string} module Module slug.
+ */
+export const getExistingTag = async( module ) => {
+
+	try {
+		let tagFound = data.getCache( module, 'existingTag', 300 );
+
+		if ( ! tagFound ) {
+			const html = await fetch( `${googlesitekit.admin.siteURL}?tagverify=1&timestamp=${Date.now()}` ).then( res => {
```

Comment:
I'm thinking to store the html code in `googlesitekit` main object as temporary cache, so it won't request again. what do you think @lucymtc ?

Reply:
If we do this I believe we would need some kind of action from the user to refresh this information in the case they remove the tag, happy to discuss further